### PR TITLE
Pathway fixes

### PR DESCRIPTION
**Pull request by pyr0gan**
- Merges the protocol explanations in <protocol>-101.md into <protocol>-setup.md by placing the protocol explanation below the setup steps
- Removes the <protocol>-101.md files
- Update Solana localstorage screenshot to current UI
- Updates all Discord links to use the new URL: https://figment.io/devchat

### CODE REVIEW

File: figment-learn/new-pathways/avalanche/avalanche-setup.md [hunk 1]
```diff
@@ -1,17 +1,112 @@
 # Set your Datahub API key
 
-Create an `.env.local` file at the root of the directory. Copy and paste the contents of the existing `.env.example` into the new file and save it to disk (you could also rename `.env.example` to `.env.local`).
+Create an `.env.local` file at the root of the directory by copying the existing `.env.example` file with the terminal command
+`cp .env.example .env.local`.
```
Comment (guillaumegaluz) [inline figment-learn/new-pathways/avalanche/avalanche-setup.md:4]: i just realized we already have this in the README https://github.com/figment-networks/learn-web3-dapp
i think we can remove it from those files

File: figment-learn/new-pathways/avalanche/avalanche-setup.md [hunk 2]
```diff
@@ -1,17 +1,112 @@
...
+The X-Chain and P-Chain use a Typescript library that supports native JavaScript cross-platform.
+
+## EVM compatibility
+
+The C-Chain is fully compatible with the EVM and provides support for any Ethereum applications as the EVM API is identical to Geth's API.
+
+## Role of the AVAX token
+
+AVAX is the Avalanche network's native token. All transactions on the Avalanche network require a fee denominated in AVAX.
+
```
Comment (guillaumegaluz) [inline figment-learn/new-pathways/avalanche/avalanche-setup.md:112]: i don't think we should have this here. it's not the right time to drop all this content on users, it's a bit mind numbing. let's remove it from now we can add it back in a different form

Comment (guillaumegaluz) [reply figment-learn/new-pathways/avalanche/avalanche-setup.md:112]: same for the other protocols

Comment (pyr0gan) [reply figment-learn/new-pathways/avalanche/avalanche-setup.md:112]: Copy that

**Pull request closed without merging**